### PR TITLE
Add Pubnative bidder documentation

### PR DESCRIPTION
**Pull request by aleksanyan**

### CODE REVIEW

File: docs/bidders/pubnative.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## Prerequisite
+Before adding PubNative as a new bidder, there are 3 prerequisites:
+- As a Publisher, you need to have Prebid Mobile SDK integrated.
+- You need a configured Prebid Server (either self-hosted or hosted by 3rd party).
+- You need to be integrated with Ad Server SDK (e.g. Mopub) or internal product which communicates with Prebid Mobile SDK.
+
+## Configuration 
+
+- app_auth_token is unique per publisher app. Please contact our account manager to obtain yours.
+zone_id should be always set to 1 (unless special use case)
```
Comment (SyntaxNode) [inline docs/bidders/pubnative.md:12]: Should zone_id have a bullet point too?

Comment (aleksanyan) [reply docs/bidders/pubnative.md]: You are right, fixed thanks.

File: docs/bidders/pubnative.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Before adding PubNative as a new bidder, there are 3 prerequisites:
+- As a Publisher, you need to have Prebid Mobile SDK integrated.
+- You need a configured Prebid Server (either self-hosted or hosted by 3rd party).
+- You need to be integrated with Ad Server SDK (e.g. Mopub) or internal product which communicates with Prebid Mobile SDK.
+
+## Configuration 
+
+- app_auth_token is unique per publisher app. Please contact our account manager to obtain yours.
+- zone_id should be always set to 1 (unless special use case)
+- bidder should be always set to "pubnative"
```
Comment (guscarreon) [inline docs/bidders/pubnative.md:13]: Do you think we could clarify that these values must be present in the `openrtb`'s json request `"ext"` field?  Maybe even explain a little further that the `zone_id` is an `int` and its value limits and the `app_auth_token` is a string. All of these are just ideas, of course.

File: docs/bidders/pubnative.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# Pubnative Bidder
+
+## Prerequisite
+Before adding PubNative as a new bidder, there are 3 prerequisites:
+- As a Publisher, you need to have Prebid Mobile SDK integrated.
+- You need a configured Prebid Server (either self-hosted or hosted by 3rd party).
+- You need to be integrated with Ad Server SDK (e.g. Mopub) or internal product which communicates with Prebid Mobile SDK.
```
Comment (guscarreon) [inline docs/bidders/pubnative.md:7]: Are there any Pubnative help resources for people to consult in case they want to implement these?

Comment (aleksanyan) [reply docs/bidders/pubnative.md:7]: Added, txs.

File: docs/bidders/pubnative.md (hunk 2 above)
Comment (aleksanyan) [reply docs/bidders/pubnative.md]: Update, PTAL 👍

File: docs/bidders/pubnative.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+## Testing
+
+Please consult with our Account Manager for testing. 
+We need to confirm that your ad request is correctly received by our system.
+
+The following test parameters can be used to verify that Prebid Server is working properly with the 
+Appnexus adapter.
+
+The following json can be used to do a request to prebid server for verifying it's integration with Pubnative adapter.
```
[the remark this answers is not in the document]
Comment (aleksanyan) [reply docs/bidders/pubnative.md]: fixed

**Pull request merged**